Doc/whatsnew/3.9.rst

```diff
@@ -125,8 +125,15 @@ that schedules a shutdown for the default executor that waits on the
 :func:`asyncio.run` has been updated to use the new :term:`coroutine`.
 (Contributed by Kyle Stanley in :issue:`34037`.)
 
+fcntl
+-----
+
+:data:`~fcntl.F_OFD_GETLK`, :data:`~fcntl.F_OFD_SETLK` and
+:data:`~fcntl.F_OFD_SETLKW` are added for fcntlmodule.
```

Comment:
There is no "fcntlmodule". And mentioning the "fcntl" module would be redundant. I suggest:
```
Added constants :data:`~fcntl.F_OFD_GETLK`, :data:`~fcntl.F_OFD_SETLK` and :data:`~fcntl.F_OFD_SETLKW`.
```